routers/web/user/setting/actions.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package setting
+
+import (
+	"fmt"
+	"net/http"
+
+	"code.gitea.io/gitea/modules/base"
+	"code.gitea.io/gitea/modules/context"
+)
+
+const (
+	tplSettingsActions base.TplName = "user/settings/actions"
+)
+
+func Actions(ctx *context.Context) {
+	ctx.Data["Title"] = ctx.Tr("actions.actions")
+	ctx.Data["PageIsSettingsActions"] = true
+	ctx.Data["RunnersBaseLink"] = fmt.Sprintf("%s/runners", ctx.Link)
+	ctx.Data["SecretsBaseLink"] = fmt.Sprintf("%s/secrets", ctx.Link)
+	GetSecrets(ctx)
```

Comment:
This function name reads strange to me.

What about "prepareSecretsData" ?

Reply:
https://github.com/go-gitea/gitea/pull/24200/commits/26eb30487093ec015742db7cfab4557b32c45602 and https://github.com/go-gitea/gitea/pull/24200/commits/90fea6c33591f808e5a9811742eba546f3fa99f2